CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+* Voting on all matters requiring a decision from the Technical Steering Committee.
+* Resolving any technical disputes. Anybody who disagree with the resolution of a technical issue can request that the Technical Steering Committee review the issue and make a decision on it. All reasonable steps to resolve the dispute via consensus should be taken before escalating to the Technical Steering Committee. If the Technical Steering Committee Chair concludes that this has not happened, they can refer the issue back to the complainant to take further action before escalating it to the Technical Steering Committee. The decision of the Technical Steering Committee on any technical disputes will be final and all project contributors are expected to comply. If a Maintainer or Committer fails to comply with a decision of the Technical Steering Committee, the Technical Steering Committee will may take any necessary steps to enforce its decision, including replacing the Maintainer/Committer.
+* Any new project repositories must be approved by the Technical Steering Committee.
+
+## Submitting Changes
+Inbound contributions are done through [pull requests](https://github.com/open-ness/edgeapps/pulls) which include code changes, enhancements, bug fixes or new applications/features. If you are getting started, you may refer to Github's [how-to](https://help.github.com/articles/using-pull-requests/). With your contributions, we expect that you:
+
+* certify that you wrote and/or have the right to submit the pull request,
+* agree with the [Developer Certificate of Origin](http://developercertificate.org/),
+* sign-off your contribution with `Signed-off-by` tag in the commit message(s)
+* comply with OpenNESS [licensing](#license),
+* pass all Continuous Integration & Continuous Delivery (CI/CD) tools' checks,
+* test and verify the proper behaviour of your code, and
+* accompany the contribution with good quality documentation.
+
+Signing off the contribution is performed by the command `git commit --signoff` which certifies that you wrote it or otherwise have the right to pass it on as an open-source contribution.
+
+Application contributors have the implicit commitment to continually support their applications in whenever a bug is discovered or a test is broken due to the publication of a new release.
+
+It is always admired when application contributors provide end-to-end acceptance test methods.
+
+Big contributions, ideas or controversial features should be discussed with the developers' community and with the Technical Steering Committee through a Request for Comments (RFC). This is an essential early step to bring everybody to a common level of understanding so that when the final implementation is pushed, there will be common consensus of the acceptability of the contribution and smooth integration of it to the mainline. Requests for Comments should be submitted through a pull request prefixed with `[RFC]` in the title.
+
+## Contribution Acceptance Flow
```

Comment:
do we need to cover 
1. Unattended PRs will be deleted after a while? 
2. what is our position on someone enabling competition hardware e.g. Mellanox NIC, NVidia GPU etc ?

Reply:
1. It is the responsibility of PR issuers to track and ask for community's reviews. If PR stayed for extended periods of time, the maintainers have the right to close obsolete/inactive PRs.
2. As long as these contributions can be maintained at the long run, they will be accepted. Contributors are responsible to maintain and continuously validate their hardware. Hit & run contributions may risk getting removed from the project.